package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "web-eid-webextension",
-  "version": "2.0.0",
+  "version": "2.0.1",
```

No need to increment the version yet. Version 2.0.0 has not bee officially published.
```suggestion
  "version": "2.0.0",
```